app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
+  def json_response(object, status = :ok)
+    render json: object, status: status
+  end
+end
```

Comment:
trailing whitespace - add a newline at the end of the file to get rid of the weird symbol